modules/check_in/spec/request/v2/sessions_request_spec.rb

```diff
@@ -150,6 +150,8 @@
       end
 
       it 'returns a success response' do
+        allow(Flipper).to receive(:enabled?)
+          .with('check_in_experience_services_refactor').and_return(false)
         allow_any_instance_of(::V2::Lorota::Service).to receive(:token_with_permissions).and_return(service_resp)
         expect_any_instance_of(::V2::Lorota::Service).to receive(:token_with_permissions).once
```

Comment:
Same as above comments

Reply:
addressed in 6ea726dd366a3daf29ab012fc161ed7dd6a090f7

Reply:
Could you please move these lines inside before block as well?

Reply:
addressed in c9bbe5f